drivers/clock_control/clock_control_mcux_ccm.c

```diff
@@ -128,6 +128,13 @@ static int mcux_ccm_get_subsys_rate(const struct device *dev,
 		}
 	} break;
 #endif
+
+#if CONFIG_COUNTER_MCUX_GPT
```

Comment:
Please replace `#if` with `#ifdef` for consistency with the rest of the file

Reply:
ok. done